documentation/markdown/architecture/core.md

```diff
@@ -16,9 +18,9 @@ If an error gets thrown here it means there is an issue with the input.
 For example, if the input data claims to be JSON-LD but is actually not.
 
 The power of using this interface really shines when using certain utility classes.
-The one we use the most is the `WaterfallHandler`,
+The one we use the most is the `StatusWaterfallHandler`,
 which takes as input a list of handlers of the same type.
-The input and output of a `WaterfallHandler` is the same as those of its inputs,
+The input and output of a `StatusWaterfallHandler` is the same as those of its inputs,
 meaning it can be used in the same places.
```

Comment:
This doesn't quite make sense. Maybe the first line is meant to say —
```
The outputs of a `StatusWaterfallHandler` are the same as those of its inputs,
```
The second line doesn't make sense at all, because it's suggesting that —
```
a `StatusWaterfallHandler` can be used in the same places as a `StatusWaterfallHandler`
```

Reply:
@TallTed would this be clearer?

> A `StatusWaterfallHandler` will conform to the same interface as its input handlers,
> meaning it can be used in the same places.

Reply:
@joachimvh — Maybe this?
```suggestion
The output(s) of a `StatusWaterfallHandler` will conform to the same interface(s)
as its input handler(s), meaning it can be used in the same places.
```